parsec/parsec_mpi_funnelled.c

```diff
@@ -100,6 +100,7 @@ static int mpi_funnelled_tag_unregister_unsafe_internal(parsec_ce_tag_t tag);
  * spots in the request array.
  */
 #define MAX_DYNAMIC_REQ_RANGE 30 /* according to current implementation */
+#define MAX_NUM_RECV_REQ_IN_ARRAY 15
 #define EACH_STATIC_REQ_RANGE 5 /* for each registered tag */
```

Comment:
It doesn't need to be done in this PR but it would be great if these were mutable MCAs